src/components/views/elements/Flair.js

```diff
@@ -250,6 +274,12 @@ export default class Flair extends React.Component {
 
 Flair.propTypes = {
     userId: PropTypes.string,
+
+    // Whether to show only the flair associated with related groups of the given room,
+    // or all flair associated with a user.
+    showRelated: PropTypes.bool,
```

Comment:
I would probably have just made supplying the room ID param filter the flair to that room's related groups & not bother with `showRelated`, but whatever.

Reply:
I thought about that, but then I thought we'd probably end up passing the `roomId` if the we're in the `MemberInfo`. I'm not sure if the `MemberInfo` will make use of it though, so perhaps an early optimisation. I can imagine some UI that separates "this room's flair" with "all the flair of the user" but still displays them both? Anyway, if we end up not needing both, we can change it later.